modules/nlu/src/backend/pipelines/intents/svm_classifier.ts

```diff
@@ -149,7 +165,12 @@ export default class SVMClassifier {
       }
 
       const svm = new this.toolkit.SVM.Trainer({ kernel: 'LINEAR', classifier: 'C_SVC' })
-      await svm.train(l1Points, progress => debugTrain('SVM => progress for INT', { context, progress }))
+
+      await svm.train(l1Points, progress => {
+        this.realtime.sendPayload(this.realtimePayload.forAdmins('statusbar.event', getPayloadForProgress(progress)))
```

Comment:
this will make the bar progress from 0 to 100 multiple times right ?

Ideally we would have a single bar that trains one time, depending on the progress of each `l1` and CRF.

Reply:
This is not making it progressing multiple times, but it is still a mistake. I will fix this.